code/modules/mob/living/carbon/alien/adult/caste/hunter.dm

```diff
@@ -75,11 +75,10 @@
 					blocked = TRUE
 			if(!blocked)
 				L.visible_message(span_danger("[src] pounces on [L]!"), span_userdanger("[src] pounces on you!"))
-				L.Paralyze(100)
-				sleep(0.2 SECONDS)//Runtime prevention (infinite bump() calls on hulks)
+				L.Knockdown(10 SECONDS)
 				step_towards(src,L)
 			else
-				Paralyze(40, ignore_canstun = TRUE)
+				Knockdown(4 SECONDS)
```

Comment:
you dont understand what this piece of code does. this doesnt work

Reply:
did we change the knockdown macros?

Reply:
no. you are trying to knockdown an alien hunter. a mob that by default cant be stunned

Reply:
this is not a problem with my code, this is a problem with xenomorphs being knockdown immune

Reply:
it is a problem with your code because youre making it not work